src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java

```diff
@@ -0,0 +1,75 @@
+package com.google.checkstyle.test.chapter5naming.rule53camelcase;
+
+class InputCamelCaseDefined {
+
+    int newCustomerId;
+
+    String innerStopwatch;
+
+    boolean supportsIpv6OnIos;
+
+    void XmlHttpRequest() {}
+
+    void YouTubeImporter() {}
+
+    void YoutubeImporter() {}
+
+    class InnerGood {
+
+        int newCustomerId;
+
+        String innerStopwatch;
+
+        boolean supportsIpv6OnIos;
+
+        void XmlHttpRequest() {}
+
+        void YouTubeImporter() {}
+
+        void YoutubeImporter() {}
+    }
+
+        InputCamelCaseDefined anonymousGood
+            = new InputCamelCaseDefined() {
+
+            int newCustomerId;
+
+            String innerStopwatch;
+
+            boolean supportsIpv6OnIos;
+
+            void XmlHttpRequest() {}
+
+            void YouTubeImporter() {}
+
+            void YoutubeImporter() {}
+    };
+}
+
+class AbbreviationsIncorrect {
+
+    int newCustomerID; // violation 'Abbreviation .* 'newCustomerID' .* '1' .*'
```

Comment:
too cryptic
https://checkstyle.org/checks/naming/abbreviationaswordinname.html#Violation_Messages we have single message 
`Abbreviation in name ''{0}'' must contain no more than ''{1}'' consecutive capital letters.`

can we do `no more than ''1'' consecutive capital letters.` ?
or `no more than ''1'' .* capital letters.`

Reply:
Tried this:

https://github.com/checkstyle/checkstyle/blob/ddd818855aa6427ba31af0dc895c4f260b04bc38/src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java#L78-L82

it is close to cross the 100 width column limit